apps/extension/src/routes/popup/settings/settings-connected-sites/known-site.tsx

```diff
@@ -44,14 +59,23 @@ export const KnownSite = ({
         </span>
       )}
 
-      <div className='flex items-center gap-2'>
-        {site.choice === UserChoice.Approved && (
-          <FrontendUrlButton
-            isSelected={frontendUrl === site.origin}
-            onClick={() => setFrontendUrl(site.origin)}
-          />
-        )}
+      {site.choice === UserChoice.Approved && (
+        <TooltipProvider>
+          <Tooltip>
+            <TooltipTrigger
+              className='ml-auto cursor-pointer border-0 bg-transparent'
+              onClick={onToggleDefault}
+            >
+              {isDefaultFrontend ? <StarFilledIcon /> : <StarIcon />}
+            </TooltipTrigger>
+            <TooltipContent>
+              {isDefaultFrontend ? 'Primary Penumbra frontend' : 'Set as primary Penumbra frontend'}
+            </TooltipContent>
+          </Tooltip>
+        </TooltipProvider>
+      )}
```

Comment:
Is this how it's intended that we can set the default frontend through both the` connected sites` settings and the `default frontend` setting?

Reply:
Misunderstood this part of the task. Actually, the stars were meant to be removed from this page, so the "Default frontend" settings page would be the only entry point into this